src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/experimental_detectron_topk_rois.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <vector>
+#include "single_layer_tests/experimental_detectron_topkrois.hpp"
+
+using namespace ov::test;
+using namespace ov::test::subgraph;
+
+namespace {
+std::vector<int64_t> maxRois {
+        1000,
+        1500,
+        2000
+};
+
+const std::vector<std::vector<InputShape>> staticInputShape = {
+        static_shapes_to_test_representation({{3000, 4}, {3000}}),
+        static_shapes_to_test_representation({{4200, 4}, {4200}}),
+        static_shapes_to_test_representation({{4500, 4}, {4500}})
+};
+
+INSTANTIATE_TEST_SUITE_P(smoke_ExperimentalDetectronTopKROIs_static, ExperimentalDetectronTopKROIsLayerTest,
+                         ::testing::Combine(
+                                 ::testing::ValuesIn(staticInputShape),
+                                 ::testing::ValuesIn(maxRois),
+                                 ::testing::Values(ElementType::f32),
```

Comment:
Could you add f16 test cases? Looks like it's not covered by neither functional nor unit tests

Reply:
I can either rework SubgraphBaseTest here
https://github.com/openvinotoolkit/openvino/blob/ec38a88b60ea7e792e8d779d21709418c95562e7/src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp#L234 
to override presicions extraction (sth like I did it here https://github.com/openvinotoolkit/openvino/blob/a79830cb55623ce2654f0685b0ca51b3f412195a/src/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp#L403 )
or just add f16 case to unit tests only. 
What case do you prefer?

Reply:
so  it works with reference conversion also, i've just changed the inputs generation to  generate f16 inputs

Reply:
done